alpakka/src/main/scala/org/scanamo/ops/retrypolicy/RetryPolicy.scala

```diff
@@ -16,35 +16,40 @@
 
 package org.scanamo.ops.retrypolicy
 
+import java.util.SplittableRandom
 import org.scanamo.ops.retrypolicy.RetryPolicy._
-
 import scala.concurrent.duration.Duration
 
 sealed abstract class RetryPolicy extends Product with Serializable { self =>
   final def continue: Boolean = self match {
     case Max(0) | Never              => false
     case And(thisPolicy, thatPolicy) => thisPolicy.continue && thatPolicy.continue
     case Or(thisPolicy, thatPolicy)  => thisPolicy.continue || thatPolicy.continue
+    case Jitter(_, _, innerPolicy)   => innerPolicy.continue
     case _                           => true
   }
 
-  final def delay: Duration = self match {
-    case Constant(retryDelay)               => retryDelay
-    case Linear(retryDelay, n)              => retryDelay * n.toDouble
-    case Exponential(retryDelay, factor, n) => retryDelay * Math.pow(factor, n.toDouble)
-    case And(thisPolicy, thatPolicy)        => thisPolicy.delay.max(thatPolicy.delay)
-    case Or(thisPolicy, thatPolicy)         => thisPolicy.delay.min(thatPolicy.delay)
-    case Never                              => Duration.Inf
```

Comment:
@Divyanshu564 can you please check it either we need it or not as  I am bit  doubt about it?
what you think @regiskuckaertz ?

Reply:
@AyushTiwary `Never` case is required as without this case `Or(thisPolicy: RetryPolicy, thatPolicy: RetryPolicy)` will always calculate the wrong delay due to `Math.min(Math.min(thisPolicy.delay, thatPolicy.delay))`

Reply:
Yes, durations form a lattice, it is needed otherwise combinations are unsound.

Reply:
true